2022q4/openstack-on-freebsd.adoc

```diff
@@ -0,0 +1,53 @@
+=== OpenStack on FreeBSD
+
+Links: +
+link:https://www.openstack.org/[OpenStack] URL: link:https://www.openstack.org/[] +
+link:https://github.com/openstack-on-freebsd[OpenStack on FreeBSD] URL: link:https://github.com/openstack-on-freebsd[] +
+
+Contact: Chih-Hsin Chang <starbops@hey.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+OpenStack is an open-source cloud operating system for different resource types like virtual and bare-metal machines.
+Users can spawn FreeBSD instances on the open cloud platform, but it is not currently possible to run the OpenStack control plane on FreeBSD hosts.
+This project aims to port key OpenStack components so that FreeBSD can function as an OpenStack host.
+
+In 2022 Q4, we have almost completed the porting work regarding the crucial OpenStack components.
+Most of the components/services composing an essential OpenStack cluster are now able to run on FreeBSD hosts, including:
+
+* Keystone (identity service)
+** keystone server
+* Glance (image service)
+** glance-api
+* Placement (resource tracking and inventory service)
+** placement-api
+* Neutron (networking service)
+** neutron-server
+** neutron-metadata-agent
+** neutron-dhcp-agent
+** neutron-openvswitch-agent
+* Nova (compute service)
+** nova-api
+** nova-conductor
+** nova-scheduler
+** nova-compute
+** nova-novncproxy
+
+The step-by-step documents for constructing a POC can be found link:https://github.com/openstack-on-freebsd/docs[in the `docs` repository].
+
+In its design, most of the OpenStack components provide an abstraction layer for the underlying implementations.
+For nova, we choose the combination of the `libvirt` driver with the `bhyve` virtualization type enabled.
+For neutron, it is the `openvswitch` mechanism driver.
+We solved several runtime dependencies and porting issues against the Libvirt, bhyve, and Open vSwitch combinations with minimal effort.
+It indicates we still have lots of work to undertake to make the changes back to OpenStack upstream.
```

Comment:
> It indicates we still have …

With four sentences preceding the word 'It', the meaning of it is unclear. 

Instead, maybe: 

> We still have …